src/SFA.DAS.EmployerApprenticeshipsService.Application/Commands/RefreshEmployerLevyData/RefreshEmployerLevyDataCommandHandler.cs

```diff
@@ -75,6 +81,23 @@ public class RefreshEmployerLevyDataCommandHandler : AsyncRequestHandler<Refresh
             }
         }
 
+        /// <summary> 
+        /// If there are any Submissions from Hmrc that have the same submission Id, we should discard all 
+        /// but the first. 
+        /// </summary> 
+        private DasDeclaration[] FilterDuplicateHmrcDeclarations(string empRef,
+            DasDeclaration[] declarations)
+        { 
+            var duplicateIds = declarations.GroupBy(d => d.SubmissionId).Where(g => g.Count() > 1)
+                .Select(s => s.First().SubmissionId).ToList(); 
+ 
+            duplicateIds.ForEach(submissionId => 
```

Comment:
Do you think the message might make diagnosis easier if it was a single message containing the submission ids appended together into a csv string, so that we can see all the duplicated IDs in one place?